freespace2/resources.cmake

```diff
@@ -51,19 +51,14 @@ elseif(APPLE)
         COMMAND cp -a "${CMAKE_CURRENT_SOURCE_DIR}/${subpath}/Resources" "$<TARGET_FILE_DIR:Freespace2>/../Resources"
         COMMENT "Copying resources into bundle..."
     )
+    
+    CONFIGURE_FILE("${CMAKE_CURRENT_SOURCE_DIR}/${subpath}/InfoPlist.strings.in" "${CMAKE_CURRENT_BINARY_DIR}/InfoPlist.strings")
 
-    #configure_file("${CMAKE_CURRENT_SOURCE_DIR}/${subpath}/fixup_bundle.cmake.in"
-    #    "${CMAKE_CURRENT_BINARY_DIR}/fixup_bundle.cmake"
-    #   @ONLY)
-
-    #file(GENERATE
-    #    OUTPUT "${CMAKE_CURRENT_BINARY_DIR}/fixup_bundle-$<CONFIG>.cmake"
-    #    INPUT "${CMAKE_CURRENT_BINARY_DIR}/fixup_bundle.cmake")
-
+    # Put the generated files into the bundle
     add_custom_command(TARGET Freespace2 POST_BUILD
 		COMMAND cp -a "${FSO_MAC_FRAMEWORKS}" "$<TARGET_FILE_DIR:Freespace2>/../Frameworks"
-        #COMMAND "${CMAKE_COMMAND}" -P "${CMAKE_CURRENT_BINARY_DIR}/fixup_bundle-$<CONFIG>.cmake"
-        COMMENT "Copying frameworks into bundle..."
+        COMMAND cp -a "${CMAKE_CURRENT_BINARY_DIR}/InfoPlist.strings" "$<TARGET_FILE_DIR:Freespace2>/../Resources/English.lproj"
```

Comment:
The `MACOSX_PACKAGE_LOCATION` property is supposed to copy the files into the bundle as part of the build but in my test that didn't work.